src/main/java/fi/dy/masa/minihud/config/Configs.java

```diff
@@ -46,6 +47,8 @@ public static class Generic
         //public static final ConfigBoolean       FIX_VANILLA_DEBUG_RENDERERS         = new ConfigBoolean("enableVanillaDebugRendererFix", true, "If true, then the vanilla debug renderer OpenGL state is fixed.");
         public static final ConfigDouble        FONT_SCALE                          = new ConfigDouble("fontScale", 0.5, 0.01, 100.0, "Font scale factor for the info line HUD. Default: 0.5\n");
         public static final ConfigOptionList    HUD_ALIGNMENT                       = new ConfigOptionList("hudAlignment", HudAlignment.TOP_LEFT, "The alignment of the info line HUD");
+        public static final ConfigHotkey        INVENTORY_PREVIEW                   = new ConfigHotkey("inventoryPreview", "LEFT_ALT", KeybindSettings.PRESS_ALLOWEXTRA, "The key to activate the inventory preview feature");
+        public static final ConfigHotkey        INVENTORY_PREVIEW_TOGGLE_SCREEN     = new ConfigHotkey("inventoryPreviewToggleScreen", "BUTTON_3", KeybindSettings.create(KeybindSettings.Context.ANY, KeyAction.PRESS, true, true, false, true), "Open a screen for inventory preview\nYou can use your mouse to see tooltips");
```

Comment:
I need to change this default toggle key to something else, because it breaks the "normal" pick functionality of Vanilla...

Reply:
> I need to change this default toggle key to something else, because it breaks the "normal" pick functionality of Vanilla...

This toggle key should only work when holding `INVENTORY_PREVIEW` key, so it is OK. If it breaks vanilla tha is a bug, I didn't tested that.

Reply:
I made it into an "ALT + X" combo because you'd already be holding left_alt

Reply:
This is a bug, I ve fixed it in 1cf25f5